corn-maze.lic

```diff
@@ -0,0 +1,632 @@
+        custom_move "#{x}"
+      else
+        custom_move("#{x}",true)
+      end
+    end
+    echo "TURN IN TASK THEN RETURN HERE!"
+    Flags.delete('poisoned')
+    finish_task
+  end
+
+  def scarecrow_task
+    Flags.add('scarecrow_done', 'You put the finishing touches')
+    Flags.add('already_built', 'already built a scarecrow here')
+    @maze_path.each do |x|
+      exit if Flags['maze_done']
+      if !Flags['Task_done']
+        if !DRRoom.room_objs.include? "corn husk scarecrow"
+          while !Flags['scarecrow_done'] && !Flags['maze_done']
+            fput "build scarecrow"
+            pause 1
+            waitrt?
+          end
+        end
+      end
+      custom_move "#{x}"
+      Flags.reset('scarecrow_done')
+    end
+    finish_task
+  end
+
+  def mice_task
+    @mice_path.each do |x|
+      exit if Flags['maze_done']
+      if !Flags['Task_done']
+        fput 'wave'
+        pause 1
+        waitrt?
+        DRC.fix_standing
+        fput "TASK"
+        custom_move "#{x}"
+      else
+        custom_move("#{x}",true)
+      end
+    end
+    echo "TURN IN TASK THEN RETURN HERE!"
+    finish_task
+  end
+
+  def scream_task
+    @scream_path.each do |x|
+      exit if Flags['maze_done']
+      if x == 'scream'
+        wait_on_done_move
+        fput "#{x}"
+      else
+        custom_move("#{x}",true)
+      end
+    end
+    echo "TURN IN TASK THEN RETURN HERE!"
+    finish_task
+  end
+
+  def weeds_task
+    @maze_path.each do |x|
+      exit if Flags['maze_done']
+      if !Flags['Task_done']
+        fput 'pull weeds'
+        pause 1
+        waitrt?
+        while !checkstanding
+          fput 'stand'
+        end
+        DRC.fix_standing
+        fput "TASK"
+        custom_move "#{x}"
+      else
+        custom_move("#{x}",true)
+      end
+    end
+    echo "TURN IN TASK THEN RETURN HERE!"
+    finish_task
+  end
+
+  def corn_task
+    @maze_path.each do |x|
+      exit if Flags['maze_done']
+      if !Flags['Task_done']
+        DRC.bput('forage corn', 'You forage around for a moment')
+        pause 1
+        waitrt?
+      fput('drop corn') if /corn/ =~ left_hand || /corn/ =~ right_hand
+        DRC.fix_standing
+        fput "TASK"
+        custom_move "#{x}"
+      else
+        custom_move("#{x}",true)
+      end
+    end
+    echo "TURN IN TASK THEN RETURN HERE!"
+    finish_task
+  end
+
+  def tokens_task
+    @maze_path.each do |x|
+      exit if Flags['maze_done']
+      if !Flags['Task_done']
+        fput 'search'
+        pause 1
+        waitrt?
+        DRC.fix_standing
+        fput "TASK"
+        custom_move "#{x}"
+      else
+        custom_move("#{x}",true)
+      end
+    end
+    echo "TURN IN TASK THEN RETURN HERE!"
+    finish_task
+  end
+
+  def grasshopper_task
+    @mice_path.each do |x|
+      exit if Flags['maze_done']
+      if !Flags['Task_done']
+        fput 'search'
+        pause 1
+        waitrt?
+        DRC.fix_standing
+        custom_move "#{x}"
```

Comment:
💡 I'm pretty sure all of these `custom_move` calls don't need the string interpolation. `x` should already be a string